README.md

```diff
@@ -1,141 +1,101 @@
-<p align="center"><img src="https://user-images.githubusercontent.com/65415371/124739522-d8393d00-df11-11eb-9741-820955887d21.png" width="1000px" /></p>
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-<a href='https://posthog.com/contributors'><img src='https://img.shields.io/badge/all_contributors-190-orange.svg?style=flat-square' /></a>
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<p align="center">
+  <img width="300" alt="posthoglogo" src="https://user-images.githubusercontent.com/85295485/144591858-cb39e04a-7bf4-4a4d-a1fd-5d05a3ff693f.png">
+</p>
+<p align="center">
+  <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+  <a href='https://posthog.com/contributors'><img alt="Github All Contributors" src="https://img.shields.io/github/all-contributors/posthog/posthog"></a>
+  <!-- ALL-CONTRIBUTORS-BADGE:END -->
+  <a href='http://makeapullrequest.com'><img alt='PRs Welcome' src='https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=shields'/></a>
+  <a href='https://posthog.com/slack'><img alt="Join Slack Community" src="https://img.shields.io/badge/slack%20community-join-blue"/></a>
+  <img alt="Docker Pulls" src="https://img.shields.io/docker/pulls/posthog/posthog"/>
+  <img alt="GitHub commit activity" src="https://img.shields.io/github/commit-activity/m/posthog/posthog"/>
+  <img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/posthog/posthog"/>
+</p>
+
+## PostHog is an open-source product analytics suite, built for engineers
+* Automatically track every event on your website or app
+* Understand your users and how to improve your product
+* Deploy on your own infrastructure to keep control of your data.
 
-<br />
 
-PostHog is an open-source product analytics suite, built for developers. Automate the collection of every event on your website or app, with no need to send data to third-parties. 
-
-You can deploy on your own infrastructure, and PostHog plays nicely with data warehouses through plugins for import, transformation, and export. 
-
-<br /><br />
-
-<img align="left" src="https://user-images.githubusercontent.com/65415371/124739629-f43cde80-df11-11eb-9033-c5d1d7194f03.png" width="50px" />
-
-## Quick start
-
-It's easy and free to get started.
+## Get started for free
 
-<a href="https://posthog.com/docs/self-host/overview#deploy">Deploy PostHog Self-Hosted</a> in your existing infrastructure.
+### Option 1: Hobby instance one-line-deploy
+For <100K events on Linux with Docker (recommended 4GB memory):
 
-... or use [PostHog Cloud](https://app.posthog.com/signup) if you don't need to self-host.
+ ```bash 
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/deploy-hobby)" 
```

Comment:
@fuziontech are you cool with this bring front and centre on our Readme?

I've given out the hobby instance a good go, it was fast to set up on DO and seems to be running just great with around 3M events, ingested over 3 days.

Are there any other limitations we should state here before bringing too much attention to it?